src/index.js

```diff
@@ -1,48 +1,77 @@
 import './style.css';
+/* eslint-disable no-console */
+/* eslint-disable arrow-body-style */
+const API = {
+  createGame: async () => {
+    return { gameId: 'scores' };
+  },
+  getScores: async (gameId) => {
+    const scores = JSON.parse(localStorage.getItem(gameId)) || [];
+    return { scores };
+  },
+  submitScore: async (gameId, name, score) => {
+    const scores = JSON.parse(localStorage.getItem(gameId)) || [];
+    scores.push({ name, score });
+    localStorage.setItem(gameId, JSON.stringify(scores));
+    return { scores };
+  },
+};
```

Comment:
- [ ] Please from the project requirement, you should read and use the [Leaderboard API documentation](https://www.notion.so/microverse/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3) and make sure that you send and receive data from/to the [Leaderboard API](https://www.notion.so/microverse/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3) but not the `localStorage` :smile: So make sure that you use `async` and `await` with the `fetch` method to send and receive data from the Leaderboard API.